guides/_subsections/automated-analysis.md

```diff
@@ -21,67 +21,43 @@ for use in the *Dashboard*. Read the section on the
   <li>
     {% include_relative _subsections/common/select-a-recording.md
       select-a-recording-caption="
-        If you do not have any recordings present in the Active Recordings
+        If you do not have any <code>Recordings</code> present in the <i>Active Recordings</i>
         view, follow
-        <a href='#startstop-a-recording'>Start/Stop a Recording</a>
+        <a href='#startstop-a-recording'>Start/Stop a <code>Recording</code></a>
```

Comment:
Same thing as above. The Start/Stop here isn't even italicized either :laughing: